Library/Homebrew/cmd/update-report.rb

```diff
@@ -167,16 +170,113 @@ def migrate_legacy_cache_if_necessary
     end
   end
 
-  def link_completions_and_docs
-    return if HOMEBREW_PREFIX.to_s == HOMEBREW_REPOSITORY.to_s
+  def migrate_legacy_repository_if_necessary
+    return unless HOMEBREW_PREFIX.to_s == "/usr/local"
```

Comment:
Is it possible someone could have their repo at `/usr/local` but not their prefix?